x-pack/legacy/plugins/apm/server/lib/metrics/by_agent/default.ts

```diff
@@ -7,12 +7,13 @@
 import { Setup } from '../../helpers/setup_request';
 import { getCPUChartData } from './shared/cpu';
 import { getMemoryChartData } from './shared/memory';
+import { GenericMetricsChart } from '../transform_metrics_chart';
 
 export async function getDefaultMetricsCharts(
   setup: Setup,
   serviceName: string
 ) {
-  const charts = await Promise.all([
+  const charts: GenericMetricsChart[] = await Promise.all([
```

Comment:
Why do we need to type this? Everything seemingly works fine when I remove it.

Reply:
Did you run typescript for the entire project? I think it causes issues downstream where it expects `GenericMetricsChart` but receives something else.

Reply:
I'll double check to see if it's still the case though.